errors/types.go

```diff
@@ -100,6 +100,14 @@ func (ce CommonEdgeX) DebugMessages() string {
 
 // Message returns the first level error message without further details.
 func (ce CommonEdgeX) Message() string {
+	if ce.message == "" && ce.err != nil {
+		if w, ok := ce.err.(CommonEdgeX); ok {
+			return w.message
```

Comment:
Shouldn't this be `w.Message()` so it is recursive.

Reply:
thanks, @lenny-intel 
my bad, and I added one more unit test to verify the recursive